CorsixTH/Src/th_map.h

```diff
@@ -423,6 +423,16 @@ class THMap
     bool* m_pPurchasableMatrix;
 };
 
+//! Add or remove divider wall for the given node
+/*!
+    If the given 'pNode' has an indoor border to another parcel in the 'delta' direction:
+    * A divider wall is added in the layer specified by 'block' if the owners
+      of the two parcels are not the same, or
+    * A divider wall is removed if the owners are the same and 'iParcelId' is involved.
+*/
+static void addRemoveDividerWalls(THMap* pMap, THMapNode* pNode, const THMapNode* pOriginalNode,
```

Comment:
I see. I was thinking that maybe all documentation for functions should be in .h